dotnet-engine/Yggdrasil.Engine.Tests/Utils.cs

```diff
@@ -0,0 +1,18 @@
+using System.Collections.Generic;
+
+namespace Yggdrasil.Test;
+
+public class CustomStrategyReturningTrue : IStrategy
+{
+    public CustomStrategyReturningTrue(string name)
+    {
+        Name = name;
+    }
+
+    public string Name { get; private set; }
```

Comment:
Nit: Can we move this to the top so we can have methods together and fields together?

Reply:
Done!